data/174/641/506/1/1746415061.geojson

```diff
@@ -0,0 +1,154 @@
+{
+  "id": 1746415061,
+  "type": "Feature",
+  "properties": {
+    "date:inception_lower":"1970-01-01",
+    "date:inception_upper":"1970-12-31",
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"1970",
+    "eu-cdda:cddaCountr":"FR",
+    "eu-cdda:cddaId":660,
```

Comment:
Same comment as in the UK... if this ID is only unique for the layer not the provider than the source prefix needs to be more specific, please.

Reply:
This is a different source than the UK data, but these are unique codes to the whole CDDA database.. so I think we're good here?

Reply:
Agree we're good here for EU data. My remaining comment is just for UK.